plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformTelemetryManager.kt

```diff
@@ -6,8 +6,13 @@ package software.aws.toolkits.jetbrains.services.codemodernizer
 import com.intellij.openapi.components.Service
 import com.intellij.openapi.components.service
 import com.intellij.openapi.project.Project
+import io.ktor.network.sockets.connect
```

Comment:
What is this import?

Reply:
Whoops don't know how that got in there, removing now